jmix-ui/ui/src/main/resources/io/jmix/ui/screen/layout.xsd

```diff
@@ -4234,6 +4234,7 @@
                     <xs:element name="properties" minOccurs="0">
                         <xs:complexType>
                             <xs:attribute name="include" type="xs:string" use="required"/>
+                            <xs:attribute name="includeProperties" type="xs:string"/>
```

Comment:
I think it is okay to keep `include` as required as the FilterPropertiesInspector.test() makes use of it.